Gemfile

```diff
@@ -1,8 +1,11 @@
-source 'http://rubygems.org'
+source "https://rubygems.org"
+
 gem "gosu"
```

Comment:
```suggestion
gem "gosu"

gem "rake", group: [:test, :dev]
```

Reply:
And then also run `bundle` and add the Gemfile.lock to Git, so that GitHub Actions and other developers will use the same version of all gems.

Reply:
Oh I see, I'll add it now